### PR TITLE
Rewrite gap filling queries to not use CTE

### PR DESCRIPTION
**Pull request by svenklemm**
This modifies the 2 gap filling queries in the docs section to not use CTEs.

### CODE REVIEW

File: using-timescaledb/reading-data.md [hunk 1]
```diff
@@ -282,31 +277,29 @@ This query will then output data in the following form:
  2017-09-22 |   9855
 ```
 Unlike with date_trunc, we can use a custom time interval with the time_bucket function,
-but in order to do gap filling properly with it we will also need to use time_bucket on our generated series.
+but in order for the generates times to align with the subselect we need use time_bucket on
```
Comment (solugebefola) [inline using-timescaledb/reading-data.md]: "generates" >> "generated"

Comment (solugebefola) [reply using-timescaledb/reading-data.md]: "...we need _to_ use..."

File: using-timescaledb/reading-data.md [hunk 2]
```diff
@@ -282,31 +277,29 @@ This query will then output data in the following form:
...
+the first argument to generate_series.
 For example, let's say you want 1080 data points in the last two weeks, and as many graphing
```
Comment (solugebefola) [inline using-timescaledb/reading-data.md]: Not part of this PR, but comma should go from in front of "and" to after, I believe.

File: using-timescaledb/reading-data.md [hunk 3]
```diff
@@ -282,31 +277,29 @@ This query will then output data in the following form:
...
 generate the correct timestamp for each of the data points even if there is no data there.
 Note that we can do basic arithmetic operations on intervals easily in order to get the correct
 value to pass to time_bucket.
 ```sql
-WITH data AS (
+SELECT
+  period AS time,
+  avg_data
+FROM
+  generate_series(time_bucket(interval '2 weeks' / 1080, now()-'2 weeks'::interval), now(), '2 weeks'::interval / 1080) AS period
```
Comment (davidkohn88) [inline using-timescaledb/reading-data.md]: Add a new line to distinguish between time_bucket call and generate series call.

Comment (davidkohn88) [reply using-timescaledb/reading-data.md]: We might also want to use named arguments? (Just for clarity?)
